lib/adal.js

```diff
@@ -147,6 +147,9 @@ AuthenticationContext = function (config) {
 
     this.config = this._cloneConfig(config);
 
+    // by default we need to move to LOGIN_REQUEST url after window callback handling.
+    this.config.navigateToLoginRequestUrl = true;
```

Comment:
Since navigateToLoginRequestUrl is defined on the config object, setting it to true like this in the constructor, will overwrite the value passed by the user in the config object when the constructor is called.
Could you please change it to something like:
if (typeof this.config.navigateToLoginRequestUrl === 'undefined')
            this.config.navigateToLoginRequestUrl = true;
By doing this, if the value is undefined, we set it to true or else we use whatever value is passed to us by the user.